.bit/responses/3.2-Week 3 Step 2.md

```diff
@@ -1,109 +1,75 @@
-    id: config.databaseId
-});
-
-const { container } = await client
-    .database(config.databaseId)
-    .containers.createIfNotExists(
-        { id: config.containerId, config.partitionKey },
-        { offerThroughput: 400 }
-    );
-}
-```
-
-5. We've finished our `create` function! Now, we can export it.
-```js
-export default create;
-```
+## **:pencil: Task 2: Call the Giphy API using the translate endpoint.**
 
-Your `databaseContext.js` file should now look like:
+_Note: **Node-fetch** must be installed for the code in this step to work! It should already be installed, but you need to require it in the first lines of code._
 
-```js
-const config = require("./config");
-const CosmosClient = require("@azure/cosmos").CosmosClient;
+Once you are done, the function should call the Giphy API and **return the data** for a gif that matches the dominant emotion from your emotion data.
 
-async function create(client, databaseId, containerId) 
-{
-   const { database } = await client.databases.createIfNotExists({
-      id: config.databaseId
-   });
+<details>
+<summary>:exclamation: Can I get a more detailed list of steps?</summary>
+  </br>
+  
+1. In the **async function**, create a new constant that will recieve the dominant emotion from your emotion data (i.e. angry, happy). This dominant emotion should be determined before you call on the trigger function.
+2. Using **node-fetch**, call the Giphy **translate endpoint** from the documentation.
+3. Run the constant through the **string** parameter in the endpoint (look as the documentation if this sounds confusing). Add your **Giphy API key**, and set the number of returned gifs to 1 by adding `&limit=1` to the endpoint.
+4. Declare another constant that formats the results of the API call in **json** format.
+5. Make this second constant the **output** of the function in `context.res`.
 
-   const { container } = await client
-      .database(config.databaseId)
-      .containers.createIfNotExists(
-         { id: config.containerId, config.partitionKey },
-         { offerThroughput: 400 }
-      );
-   }
-}
+  <br><br/>
+  
+</details>
 
-export default create;
+<details>
+<summary>:exclamation: I'm stuck. Can I get some hints?</summary>
+  </br>
+  
+Fill in the gaps:
+  
 ```
-
-
-
-## **:pencil: Task 2: Set up your Azure Cosmos DB configuration by creating the `config.js` and `databaseContext.js` files!**
-
-
-## **:camping: Make sure to commit ONLY `databaseContext.js` (NOT `config.js`) to the root directory of a new branch `week3` in this repository to move on!**
-
-Remember that you should NEVER be committing and pushing your subscription key into GitHub. Leave your `config.js` file out of the push. See this [link](https://docs.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository) to see how to remove unwanted files from a repo's history.
\ No newline at end of file
+module.exports = async function (context, req) {

```

Comment:
Make sure this has color syntax. To create colored syntax for javascript, use this: ```js.

Example
```js
console.log("Now this is color coded!")
```